src/DeviceInterface.js

```diff
@@ -37,26 +37,14 @@ class InterfacePrototype {
         this.setupAutofill()
     }
     // Default setup used on extensions and Apple devices
-    setupAutofill ({shouldLog} = {shouldLog: false}) {
```

Comment:
Had to move this back inside the devices definitions because they are no longer using the same flow.